extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/handler/ZonePlayerHandler.java

```diff
@@ -132,6 +134,8 @@
 
     private Map<String, String> stateMap = Collections.synchronizedMap(new HashMap<String, String>());
 
+    private List<SonosMusicService> msuicServices;
```

Comment:
I don't own or use any Sonos devices. But "msuicServices"  looks like a typo. :-)

Reply:
Good catch :-)
Ah, sorry, don't know where my head is these days... I somehow mixed up Sonos with LIFX... So forget about my testing request unless you can make your bulbs play some music 🤣

Reply:
lol